### PR TITLE
Grammar edits (Chapters 1-7)

### PR DESCRIPTION
**Pull request by kylebeggs**
This first pass I wanted to focus on readability/grammar. In an effort to not have to keep rebasing so much, I am going to submit more incremental PRs. Here is the start of the book through chapter 7. You'll notice right away some format changes for the source - I added the `.JuliaFormatter.toml` file here from `Meshes.jl` and ran it. Obviously if you wanted a different format like keeping spaces in function args we can change the formatting settings.

### CODE REVIEW

File: 01-geodata.qmd [hunk 1]
```diff
@@ -12,7 +12,7 @@ geospatial data science, we need to introduce important concepts, which
 will be the foundations of **Part II**, **Part III** and **Part IV**.
 
 In this chapter, we define **geospatial data** and introduce a
-**universal representation** for it that is ideal for geostatistical
+**universal representation** for such data which is ideal for geostatistical
```
Comment (juliohm) [inline 01-geodata.qmd]: "for it" isn't correct English? I thought it was clear from the context.

File: .JuliaFormatter.toml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+indent = 2
+margin = 120
+always_for_in = true
+always_use_return = false
+whitespace_typedefs = false
+whitespace_in_kwargs = false
+whitespace_ops_in_indices = true
+remove_extra_newlines = true
+trailing_comma = false
+normalize_line_endings = "unix"
```
Comment (juliohm) [inline .JuliaFormatter.toml]: I think the current code blocks are fine. Let's keep the original formatting and focus on text improvements. Later we can decide if formatting changes are necessary.

File: 03-geoio.qmd [hunk 3]
```diff
@@ -10,29 +10,25 @@ using GeoStats
...
 focus our energy interfacing with well-tested GIS libraries
 written in C/C++ (e.g., [GDAL](https://gdal.org/index.html),
 [GEOS](https://libgeos.org)). This is precisely what the
-JuliaGeo organization has been doing over the years, and it
-is an important agenda to bring people from other languages
-that are used to the [OGC](https://www.ogc.org/standards)
-standards.
+JuliaGeo organization has been doing over the years which
+encourages people from other languages who are used to 
+the [OGC](https://www.ogc.org/standards) standards - an important endeavor.
```
Comment (juliohm) [inline 03-geoio.qmd]: Let's keep these two paragraphs intact. I appreciate the rephrasing, but I like the older version a bit more.

File: 05-transforms.qmd [hunk 4]
```diff
@@ -28,11 +28,17 @@ table = (; a, b, c, d)
 gt = georef(table, CartesianGrid(100, 100))
 ```
 
+::: {.callout-tip}
+
+The `÷` operator in Julia performs *interger* division, so `10÷2` yields `5`. Conversely, `10/2` performs *floating point* division and yields a float, `5.0`.
+
+:::
+
```
Comment (juliohm) [inline 05-transforms.qmd]: We can omit this tip, I think it is too basic for the audience of the book.

File: Manifest.toml
Comment (juliohm) [inline Manifest.toml:1]: Let's not change the Manifest.toml in this PR. Changes to the Manifest.toml require additional checks, and we will be updating it soon with a new release of the project as well.

File: foreword.qmd [hunk 5]
```diff
@@ -3,7 +3,7 @@
 I've always felt that something was off with existing approaches
 to geospatial data science in other programming languages.
 I remember sitting in the beautiful
-[Stanford's Green library](https://en.wikipedia.org/wiki/Cecil_H._Green_Library)
+[Cecil H. Green library at Stanford](https://en.wikipedia.org/wiki/Cecil_H._Green_Library)
```
Comment (juliohm) [inline foreword.qmd]: Please preserve the informal name of the library. This is how we used to call it. The formal name can be found in the Wikipedia article.

File: preface.qmd [hunk 6]
```diff
@@ -93,7 +93,8 @@ flowchart LR
 ```
 
 The chapters were written to be read in sequence, but there is
-some flexibility on how to read the parts. I recommend reading
+some flexibility on how to read the parts. Although you could read **Parts II-IV** in any
+order, I recommend reading
```
Comment (juliohm) [inline preface.qmd]: Let's keep the original text here as well.

**Review by juliohm — CHANGES_REQUESTED** [review]
Thank you @kylebeggs for the great improvements so far. Let's just erase all changes that are related to the formatter. It will be easier to merge without worrying too much about runtime errors.

File: 01-geodata.qmd (hunk 1 above)
Comment (kylebeggs) [reply 01-geodata.qmd]: Technically, it is ok. I thought it sounded a little weird. The issue is "that". I think a combo of our lines is best:

> **universal representation** for it which is ideal for geostatistical

File: 03-geoio.qmd (hunk 3 above)
Comment (kylebeggs) [reply 03-geoio.qmd]: I think it reads a bit odd (but that is my opinion), perhaps another person can help slightly rephrase? I will revert it back for now.

File: foreword.qmd (hunk 5 above)
Comment (kylebeggs) [reply foreword.qmd]: Sure, I think my main objective here was to move the Stanford. You usually would place an adjective before noun in focus which is Green Library. So I think either you should do "I remember sitting in Stanford's beautiful [Green Library]" or "I remember sitting in the beautiful [Green Library](...) at Stanford"

**Review by juliohm — APPROVED** [review]

**Comment by juliohm** [comment]
These are really nice suggestions @kylebeggs ❤️ thank you for the improvements!

**Pull request merged**